lib/sessionClient.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const fetch = require('fetch-retry-or-die');
+const log = require('@financial-times/n-logger').default;
+const config = require('./helpers/config');
+const helpers = require('./helpers/helpers');
+const clientErrors = require('./clientErrors');
+
+const options = Object.assign({}, config.fetchOptions);
```

Comment:
I get it now :) You mean that the `options` are only used in `verify()`. True, but if the other files are any indication, we will eventually have another methods in here that's going to use the `options` :)